helm/ffc-demo-payment-service-core/values.yaml

```diff
@@ -36,7 +36,7 @@ container:
   requestCpu: 40m
   limitMemory: 120Mi
   limitCpu: 80m
-  readOnlyRootFilesystem: true
+  readOnlyRootFilesystem: false
```

Comment:
Should be returned to read only file system

Reply:
Done